config/default/user.role.stanford_staff.yml

```diff
@@ -1,11 +1,12 @@
 uuid: 110e86b4-151a-4724-8611-c6112d0378c0
 langcode: en
 status: true
-dependencies: {  }
+dependencies: null
 _core:
   default_config_hash: C2n7EnkHLysQyQE7c3q09n0J3HDC5lK94cvVPBzKWA4
 id: stanford_staff
 label: 'Stanford Staff'
 weight: -4
 is_admin: null
-permissions: {  }
+permissions:
```

Comment:
No permissions should be for this role. it is meant to be just an identifier.

Reply:
Removed the permission, it was not being used because that functionality was reimplemented in another way.